src/commands/shared/request.ts

```diff
@@ -94,13 +94,14 @@ export const request =
     }>,
     authn?: Authn,
     options?: {
+      accessMessage?: string;
       message?: "all" | "approval-required" | "none";
     }
   ): Promise<RequestResponse<T> | undefined> => {
     const resolvedAuthn = authn ?? (await authenticate());
     const { userCredential } = resolvedAuthn;
     const data = await spinUntil(
-      "Requesting access",
+      options?.accessMessage ? options.accessMessage : "Requesting access",
```

Comment:
should this be dependent on the type union? `"all" | "approval-required" | "none"`

So if we see: "approval-required" we can still say: "requesting access" 🤔

Reply:
Since `"approval-required"` is hardcoded for the ssh session request, I don't think we can use it to change the message in this case. I think the request would still have `approval-required` even if the user already had been approved in the past?

https://github.com/p0-security/p0cli/pull/137/files#diff-c16738fe5df540e2a49c613e959ebaa27241c6ce8389095b737f76c65bc44f8eL125

Reply:
I see, I see 🤔 .

Reply:
You can make it dependent like this:
```
const accessMessage = (message?: string) => {
  switch (message) {
    case "approval-required":
      return "Requesting access";
    default:
      return "Requesting access";
  }
};
```
then here
```suggestion
      accessMessage(options?.message),,
```

Reply:
I made this change, but the `"approval-required"` message type will return an accessMessage of `Checking for access in P0`.

This makes this change apply to `p0 kubeconfig` as well which I think is desirable since it's similar flow to starting an ssh session.

```
const accessMessage = (message?: string) => {
  switch (message) {
    case "approval-required":
     return "Checking for access in P0";;
    default:
      return "Requesting access";
  }
};
```